shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IXHROverride.ts

```diff
@@ -0,0 +1,47 @@
+import { IPromise } from "@nevware21/ts-async";
+
+// /** IPayloadData describes interface of payload sent via POST channel */
```

Comment:
It would be ideal if we can keep this so that Post channel and Sender can use the same definitions.

This will mean refactoring the Sender to construct and pass this updated format.

Reply:
I will do it! Thanks Nev!